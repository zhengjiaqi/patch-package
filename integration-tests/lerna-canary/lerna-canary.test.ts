import { runIntegrationTest } from "../runIntegrationTest"
runIntegrationTest({
  projectName: "lerna-canary",
  shouldProduceSnapshots: true,
})
