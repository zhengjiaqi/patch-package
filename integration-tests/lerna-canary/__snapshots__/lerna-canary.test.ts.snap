// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test lerna-canary: making patch 1`] = `
"SNAPSHOT: making patch
patch-package 0.0.0
â€¢ Creating temporary folder
â€¢ Installing @parcel/codeframe@2.0.0-nightly.137 with yarn
â€¢ Diffing your files with clean files
âœ” Created file patches/@parcel+codeframe+2.0.0-nightly.137.patch

ðŸ’¡ @parcel/codeframe is on GitHub! To draft an issue based on your patch run

    yarn patch-package @parcel/codeframe --create-issue

END SNAPSHOT"
`;

exports[`Test lerna-canary: the patch applies 1`] = `
"SNAPSHOT: the patch applies
patch-package 0.0.0
Applying patches...
@parcel/codeframe@2.0.0-nightly.137 âœ”
END SNAPSHOT"
`;

exports[`Test lerna-canary: the patch looks like this 1`] = `
"SNAPSHOT: the patch looks like this
diff --git a/node_modules/@parcel/codeframe/src/codeframe.js b/node_modules/@parcel/codeframe/src/codeframe.js
index 2bf2c1c..ef0695b 100644
--- a/node_modules/@parcel/codeframe/src/codeframe.js
+++ b/node_modules/@parcel/codeframe/src/codeframe.js
@@ -41,7 +41,7 @@ const highlightSyntax = (txt: string, lang?: string): string => {
   return emphasize.highlightAuto(txt).value;
 };
 
-export default function codeFrame(
+export default function patch-package(
   code: string,
   highlights: Array<DiagnosticCodeHighlight>,
   // $FlowFixMe
END SNAPSHOT"
`;
